### DISCUSSION TITLE
Discord link

### DISCUSSION THREAD

**Discussion by wiebekaai** [remix-run/remix#5344] [General]
Just trying to learn more but can't find the discord. The link on remix.run is broken: https://rmx.as/discord (invite invalid).

**Comment by jrestall** [comment]
You can use the 'Explore Public Servers' item on Discord's left main menu and then search for 'Remix'. Should be the first result.

**Reply by wiebekaai** [reply]
Thanks!

**Comment by MichaelDeBoey** [comment]
This is now working again
